python_modules/dagster/dagster/_daemon/backfill.py

```diff
@@ -146,51 +147,68 @@ def execute_backfill_jobs(
 
         # refetch, in case the backfill was updated in the meantime
         backfill = cast(PartitionBackfill, instance.get_backfill(backfill_id))
-        # create a logger that will always include the backfill_id as an `extra`
-        backfill_logger = cast(
-            logging.Logger,
-            logging.LoggerAdapter(logger, extra={"backfill_id": backfill.backfill_id}),
-        )
-        try:
-            evaluation_time = pendulum.now("UTC")
-            tick = instance.create_tick(
-                TickData(
-                    instigator_origin_id=_BACKFILL_ORIGIN_ID,
-                    instigator_name=backfill_id,
-                    instigator_type=InstigatorType.BACKFILL,
-                    status=TickStatus.STARTED,
-                    timestamp=evaluation_time.timestamp(),
-                    selector_id=backfill_id,
-                )
+        evaluation_time = pendulum.now("UTC")
+
+        # TODO - probably need to scope by code location. for now just put everything together for testing
+        log_key = ["backfill", backfill_id, evaluation_time.strftime("%Y%m%d_%H%M%S")]
+        with InstigationLogger(
```

Comment:
3. To get these logs available in the UI, I had to do add some special logic for how to get the message from the log record https://github.com/dagster-io/dagster/pull/21787/files#diff-a2c3875505d59d1c50f93bbb312c943044d849e91c319d5349b38f57947ff16eR58-R62 because the `_annotate_record` of the InstigationLogger doesn't seem to get applied https://github.com/dagster-io/dagster/blob/97c80ef86a5c077eafe007d0af87d7751e08a604/python_modules/dagster/dagster/_core/definitions/instigation_logger.py#L120

more homework for myself today is to figure out why this is happening so that we can ideally not have that special behavior in the GQL layer

Reply:
hmm, yeah, our logger overrides is really clunky.  It's been a while, but I think it's because we want to use the Logger interface for something that's really a LogHandler.